klaus/highlighting.py

```diff
@@ -34,17 +36,18 @@ def _format_lines(self, tokensource):
 
     def _lookup_ctag(self, token):
         matches = list(self._get_all_ctags_matches(token))
-        best_matches = self.get_best_ctags_matches(matches)
-        if not best_matches:
+        try:
+            best_match = next(self.get_best_ctags_matches(matches))
```

Comment:
IMO this makes things more difficult to understand. I'd simply use a list comprehension in `get_best_ctags_matches `, so we're sure we always get a list here, and then use the `[0]` approach, which makes it very easy to see that we're only looking at the first element.